Tools/jit/build.py

```diff
@@ -22,7 +22,30 @@
         "-v", "--verbose", action="store_true", help="echo commands as they are run"
     )
     args = parser.parse_args()
-    args.target.debug = args.debug
-    args.target.force = args.force
-    args.target.verbose = args.verbose
-    args.target.build(pathlib.Path.cwd(), comment=comment)
+
+    if len(args.target) == -1:
+        args.target.debug = args.debug
+        args.target.force = args.force
+        args.target.verbose = args.verbose
+        args.target.build(pathlib.Path.cwd(), comment=comment)
+
+    else:
+        # Multiple triples specified, assume this is a macOS multiarchitecture build
```

Comment:
This bit is not ideal, multiple files are created and that likely breaks dependency handling in the Makefile.

It should be possible to publish everything in a single header file, but that requires more significant changes to the builder.